Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -54,36 +44,38 @@ public override void OnPreSceneQuery()
                 Rays = new RayStep[1];
             }
 
+            closestNormal = Rotation * Vector3.forward;
+
             // Check proximity
-            NearInteractionTouchable newClosestTouchable = null;
+            BaseNearInteractionTouchable newClosestTouchable = null;
             {
-                closestDistance = distFront; // NOTE: Start at distFront for cutoff
-                foreach (var prox in NearInteractionTouchable.Instances)
+                closestDistance = float.PositiveInfinity;
```

Comment:
With this change, it looks like there will always be a `BaseNearInteractionTouchable` within "touching" distance. Previously, using `distFront` as the cutoff, we'd only determine the closest touchable object if the pointer was within a specific distance from it (which means there might be no closest touchable in many cases). Were we missing cases with that method?

Reply:
An example of a usage of `distFront` that we should make sure is accounted for, if we remove it:
https://github.com/microsoft/MixedRealityToolkit-Unity/issues/4640#issuecomment-497514917

Reply:
@keveleigh actually the comment in that issue was not that they are using distFront explicitly -- it was more like they had undone their original workaround to go back to the default value. I'm actually not clear how their original workaround got them to fix the issue they were having, to be honest.

Reply:
Ohh interesting. Thanks for the clarification!

Reply:
However, I think you do have a point Kurtis that I'd be curious to hear Niall's response to. I think this can go in and we can solve it in a separate issue, just in the interest of moving things forward. I'm sure once this is in we will discover other bugs.

Reply:
Good point, I think I actually intended on trying something here that I didn't get around to. I've created an issue and assigned it to myself #4699.